src/main/webapp/app/modules/login/login.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Redirect, RouteComponentProps } from 'react-router-dom';
+import { Grid, View } from '@adobe/react-spectrum';
+import { IRootState } from 'app/shared/reducers';
+import { login } from 'app/shared/reducers/authentication';
+import LoginForm from './login-form';
+import LoginFooter from './login-footer';
+import LoginHeader from './login-header';
+// TODO create login page image
+export interface ILoginProps extends StateProps, DispatchProps, RouteComponentProps<{}> {}
+
+export const Login: React.FC<ILoginProps> = props => {
+  const handleLogin = (username, password, rememberMe = false) => {
+    props.login(username, password, rememberMe);
+  };
+
+  return !props.isAuthenticated ? (
+    <Grid areas={['image login']} columns={['1fr', '2fr']} rows={['auto']} minHeight={window.innerHeight} data-testid="login-container">
+      {/* <Image src="https://i.imgur.com/Z7AzH2c.png" alt="alt-text" objectFit="cover" gridArea="image" />*/}
```

Comment:
Is this comment needed?

Reply:
Yes, i will uncomment this once we are ready with the login screen image and remove the next line. We already have this in backlog